app/(site)/temp/all-games/AllGamesDisplay.tsx

```diff
@@ -5,7 +5,7 @@ import Image from "next/image";
 import Game from "@/types/models/Game";
 import WishlistRow from "@/types/models/WishlistRow";
 import WishListButton from "@/app/ui/components/buttons/WishListButton";
-// import generatePhotoPid from "@/utils/generatePhotoId";
```

Comment:
I would not remove this, we might need it for later. We should only remove at the end.

Reply:
added back